quick_start.md

```diff
@@ -67,31 +67,26 @@ Local torch tensor on rank  1 :  tensor([5, 6, 7, 8, 9], dtype=torch.int32)
 
 ## New Contributors
 
-1. [Fork](https://docs.github.com/en/get-started/quickstart/contributing-to-projects) or, if you have write access, clone the [Heat repository](https://github.com/helmholtz-analytics/heat).
+1. Pick an Issue you'd like to work on. Check out [Good First Issues](https://github.com/helmholtz-analytics/heat/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22), start from most recent. Feel free to assign to yourself.
```

Comment:
Can externals assign themselves? When I wanted to assign Saj to the FFT, he had to comment first in order to appear in the list of people I could assign... it seems that he could not assign himself and that I couldnt assign externals who did not already appear in the issue (e.g. by commenting etc.)

Reply:
```suggestion
1. Pick an Issue you'd like to work on. Check out [Good First Issues](https://github.com/helmholtz-analytics/heat/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22), start from most recent. Get in touch and ask to be assigned to the issue.
```

Reply:
Good catch, addressed, thanks Fabian!